client/src/components/Admin/Users/List/index.js

```diff
@@ -19,42 +23,73 @@ const UserList = ({ users, toggleCanVote }) => {
   const usersRegistered = userKeys
     .filter(u => users[u].completedRegistration)
     .length;
-  return (
-    <table className={css.list}>
-      <thead>
-        <tr>
-          <th className={css.left}>Bruker ({totalUsers})</th>
-          <th className={css.right} title="Har fullført oppmøteregistrering">
-            Registrert ({usersRegistered})
-          </th>
-          <th className={css.right} title="(Antall med stemmerett)">
-            Rettigheter ({usersHasPermsToVote})
-          </th>
-          <th className={css.right}>
-            Stemmeberettigelse ({usersCanVote}/{usersHasPermsToVote})
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        {Object.keys(users)
-          .sort((a, b) => users[a].name.localeCompare(users[b].name))
-          .map((key) => {
-            const user = users[key];
-            return (<UserContainer
-              name={user.name}
-              canVote={user.canVote}
-              completedRegistration={user.completedRegistration}
-              key={user.id}
-              permissions={user.permissions}
-              registered={user.registered}
-              toggleCanVote={toggleCanVote}
-              id={user.id}
-            />);
+
+  const data = [...Object.keys(users)
+    .sort((a, b) => users[a].name.localeCompare(users[b].name))
+    .map(key => users[key])
+    .map(user => ({
+      name: user.name,
+      canVote: user.canVote,
+      registered: {
+        completedRegistration: user.completedRegistration,
+        registeredDate: user.registered,
+      },
+      id: user.id,
+      permissions: user.permissions,
+      setPermissions: user.setPermissions,
+    }))];
+
+  const columns = [{
+    Header: `Name (${totalUsers})`,
+    accessor: 'name',
+  }, {
+    Header: `Registrert (${usersRegistered})`,
+    accessor: 'registered',
+    className: css.center,
+    Cell: props =>
```

Comment:
To fix the props linting I would extract this inline component. 
IMO it's fine to just add another component to this file just as long as you don't put it inside the render function.